packages/internal/src/config.ts

```diff
@@ -57,6 +60,9 @@ const DEFAULT_CONFIG: Config = {
   browser: {
     open: true,
   },
+  cli: {
+    checkVersion: true,
+  },
```

Comment:
@peterp This PR will add automated, async background checks for redwoodjs package upgrades during CLI commands. The checks are enabled by default with an option to disable via `redwood.toml` config, shown here. 

I think this is an appropriate structure for the config. Thoughts/recommendations otherwise?